README.md

```diff
@@ -140,13 +140,8 @@ We test all combinations of:
 
 ## Known issues and Limitations
 
-- The [SendCtrlAltDel](docs/api_requests/actions.md#sendctrlaltdel) API request
```

Comment:
where this line ended up?

Reply:
this is where it is already described: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api_requests/actions.md#intel-and-amd-only-sendctrlaltdel

Reply:
thanks